web/index.html

```diff
@@ -4,8 +4,8 @@
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no, user-scalable=no">
     <title>Maunium sticker picker</title>
-
-    <link rel="modulepreload" href="src/widget-api.js"/>
+    <script src="https://unpkg.com/matrix-widget-api@1.6.0/dist/api.js"></script>
```

Comment:
External dependencies should be avoided unless absolutely necessary (and if they are necessary, they need to be bundled with esinstall like preact/htm is, not loaded from a random external url)

Reply:
Removed dependency on matrix-widget-api entirely. Replaced that with a new function `sendGIF` which basically only calls `sendSticker`.